Backend/app/rider-platform/rider-app/Main/src/API/UI/Leads.hs

```diff
@@ -0,0 +1,44 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module API.UI.Leads
+  ( API,
+    handler,
+  )
+where
+
+import qualified Domain.Action.UI.Leads as DLeads
+import Environment
+import EulerHS.Prelude
+import qualified Kernel.Types.APISuccess as APISuccess
+import Kernel.Types.Version
+import Kernel.Utils.Common
+import Servant
+
+type API =
+  "leads"
+    :> ( "create"
+           :> ReqBody '[JSON] DLeads.LeadsReq
+           :> Header "x-bundle-version" Version
+           :> Header "x-client-version" Version
+           :> Header "x-platform" Text
```

Comment:
Take this data also in request. Why to pass in header?

Reply:
okay